### DISCUSSION TITLE
CORS policy violation errors appear in console on saved page. Results in blank page instead of saved content.

### DISCUSSION THREAD

**Discussion by rach-1732** [gildas-lormeau/SingleFile#1573] [Q&A]
I used the tool to download some Facebook posts using self-extracting ZIP universal. On the HTML content settings, I have the 'set content security policy' enabled. 

When viewing the page on an online computer it loads totally fine. When I try and load it offline using the Edge browser, the page loads for a split second then the page goes blank and get this error:

Access to XMLHttpRequest at 'file:[local file path here].html' from origin 'null' has been blocked by CORS policy: Cross origin requests are only supported for protocol schemes: http, data, chrome-extension, edge, https, chrome-untrusted.

If I try to open the page using Firefox, the page doesn't load at all (just blank screen appears) and I get this in console:

Content Security Policy: Directive ‘frame-src’ has been deprecated. Please use directive ‘child-src’ instead.

**Comment by rach-1732** [comment]
Just to note as well on Edge, that after the initial CORS issue (which could possibly be from other chrome extensions I have loaded?), I also get repeated errors like this which appear more connected to the HTML file:

Access to XMLHttpRequest at 'file:///[drive name]/nw/' from origin 'null' has been blocked by CORS policy: Cross origin requests are only supported for protocol schemes: http, data, chrome-extension, edge, https, chrome-untrusted.

**Comment by gildas-lormeau** [comment]
Can you confirm the option `Network > blocked resources > scripts` is enabled?

**Comment by rach-1732** [comment]
It is now. And that solved it. Tku!

**Comment by gildas-lormeau** [comment]
You're welcome!
